### DISCUSSION TITLE
[AMD][Zen5] Non regression tests

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#513] [General]
To: Zen5 and EPYC Turin owners ; @madoverlord40 @huajian628 @paulyc 

Hello,

Can you please give a try to the on-going [development](https://github.com/cyring/CoreFreq/tree/develop) version `1.98.5` and check for any regression, especially in the computation of P-State frequencies ? 

The output of `corefreq-cli -s -n -m -n -i 1 -n -C 1 -B -n -k -n -M` will be appreciated

**Comment by huajian628** [comment]
If there is such a server, I will give it a try. Just a quick question: Is there any requirement for the OS version? Typically, it is an Ubuntu system.

**Reply by cyring** [reply]
Thanks for your answer.

_CoreFreq_ runs will all Linux since version `3.1` !

**Reply by cyring** [reply]
@huajian628
By the way, you can have a look in the Continuous Integration for Ubuntu builds.
For example the latest commit 
https://github.com/cyring/CoreFreq/actions/runs/11541265129
At the following step starts the _CoreFreq_ compilation 
https://github.com/cyring/CoreFreq/actions/runs/11541265129/job/32123055800#step:3:585

**Comment by cyring** [comment]
Dev ceased
